pkg/scheduler/framework/preemption/preemption_test.go

```diff
@@ -355,3 +359,480 @@ func TestSelectCandidate(t *testing.T) {
 		})
 	}
 }
+
+type fakeCandidate struct {
+	victims *extenderv1.Victims
+	name    string
+}
+
+// Victims returns s.victims.
+func (s *fakeCandidate) Victims() *extenderv1.Victims {
+	return s.victims
+}
+
+// Name returns s.name.
+func (s *fakeCandidate) Name() string {
+	return s.name
+}
+
+func TestPrepareCandidate(t *testing.T) {
```

Comment:
I was wondering whether we can cover three parts of this private func
- evict the victims(already covered)
- verify the waitingPod
- verify the nominatedNodes

Reply:
I tried to do this, but after looking at the code, I found that private variables cannot be injected externally. It seems difficult to implement. :disappointed_relieved:

Reply:
Then this could be a follow up to refactor the function for better testability. Since already LGTMed
/lgtm
/approve